android/src/main/kotlin/com/mapbox/maps/mapbox_maps/AttributionController.kt

```diff
@@ -0,0 +1,15 @@
+package com.mapbox.maps.mapbox_maps
+
+import com.mapbox.maps.MapView
+import com.mapbox.maps.mapbox_maps.mapping.applyFromFLT
+import com.mapbox.maps.mapbox_maps.mapping.toFLT
+import com.mapbox.maps.pigeons.FLTSettings
+import com.mapbox.maps.plugin.attribution.attribution
+
+class AttributionController(private val mapView: MapView) :
```

Comment:
nit:
maybe not necessary to pass whole mapView? only `attribution` and `context`

Reply:
I'd left as is - it's implemented similarly within all controllers